### PR TITLE
Ex sflow agent-id fix

### PR DESCRIPTION
**Pull request by kentik-rbarnes**
Correcting sflow agent-id to be local to the device

### CODE REVIEW

File: Juniper/QFX_EX-series/sflow-agent.conf
Comment (jryburn) [inline Juniper/QFX_EX-series/sflow-agent.conf:1]: I think we would want to pick one. I'm not sure our UI will pick up this "or" logic in the device onboarding workflow. @dpajin-kentik can keep me honest.

File: Nokia/bgp.conf
Comment (jryburn) [inline Nokia/bgp.conf:1]: I think this addition was in another PR. Should we remove it here? I'm not sure how this all works with branching in GitHub.

**Review by jryburn — CHANGES_REQUESTED** [review]
Couple questions before approving.

File: Nokia/bgp.conf
Comment (kentik-rbarnes) [reply Nokia/bgp.conf:1]: valid - didn't see that happen. now i need to figure out how to remove those nokia commits from this PR.

Comment (dpajin-kentik) [reply Nokia/bgp.conf:1]: you maybe just wanted to remove those additional two commits you had for nokia stuff

**Pull request closed without merging**